tools/flang1/flang1exe/dtypeutl.c

```diff
@@ -3423,9 +3430,10 @@ typedef enum {
   __QREAL16 = 41, /* F real(16) */
   __QCPLX32 = 42, /* F complex(32) */
   __POLY = 43,    /* F polymorphic variable */
+  __PROCPTR = 44, /* F procedure pointer descriptor */
 
   /* number of data types */
-  __NTYPES = 44 /* MUST BE LAST */
+  __NTYPES = 46 /* MUST BE LAST */
```

Comment:
Is this right?  Not 45?  I am just guessing.

Reply:
I'll double-check.